bc/subscription/tasks.py

```diff
@@ -53,11 +58,46 @@ def enqueue_posts_for_new_case(
     if document:
         files = get_thumbnails_from_range(document, "[1,2,3,4]")
 
-    for channel in get_channels_per_subscription(subscription.pk):
-        template = get_new_case_template(
-            channel.service, subscription.article_url
+    if initial_document is None:
+        initial_document = lookup_initial_complaint(subscription.cl_docket_id)
+
+    docket: DocketDict | None = None
+    if subscription.cl_docket_id:
+        docket = lookup_docket_by_cl_id(subscription.cl_docket_id)
+
+    date_filed: date | None = None
+    days_old = 0
+
+    date_filed_str = docket["date_filed"] if docket else None
+    if date_filed_str:
+        try:
+            date_filed = (
+                datetime.strptime(date_filed_str, "%Y-%m-%d").date()
+                if date_filed_str
+                else None
+            )
```

Comment:
the ternary if statement on this line is redundant. We're already checking the same condition on line 72. Could we simplify the code by removing one of them?